doc/development.md

```diff
@@ -163,3 +163,59 @@ To ensure consistent Markdown formatting run the following:
 # Use git-bash on Windows
 npm run prettier:text
 ```
+
+# DB Models (from API document)
+
```

Comment:
This new section is created, as draft to contains the relationship between database terms and API terms. This information is not required for API users, but it is useful for developers.